### PR TITLE
Typo fix: 'you you do' --> 'you do'

### PR DESCRIPTION
**Pull request by fureigh**

### CODE REVIEW

File: _pages/update-the-config-file/understanding-baseurl.md [hunk 1]
```diff
@@ -21,6 +21,6 @@ production it's `https://pages.18f.gov/guides-template{{page.url}}`.
 
 ### Change the `baseurl:` when serving locally
 
-If you you do change the `baseurl:` property in the `_config.yml` file,
+If you do change the `baseurl:` property in the `_config.yml` file,
```
Comment (jessieay) [inline _pages/update-the-config-file/understanding-baseurl.md:24]: have you considered changing this sentence to just read "You do you." ?

Comment (jessieay) [inline _pages/update-the-config-file/understanding-baseurl.md:24]: sorry that was not a real comment. just playing around with PRs I find in the `ruby-pull-requests` channel

Comment (fureigh) [inline _pages/update-the-config-file/understanding-baseurl.md:24]: "If you do you, change the `baseurl:` property in the `_config.yml` file."

you-do-you-haters-gonna-hate

Comment (jessieay) [inline _pages/update-the-config-file/understanding-baseurl.md:24]: Motion to replace all 18F docs with the sentence "You do you."

Comment (fureigh) [inline _pages/update-the-config-file/understanding-baseurl.md:24]: And to replace "Before you ship" with "Before you do you."

**Comment by fureigh** [comment]
...but do you wanna merge it, @jessieay?

**Comment by jessieay** [comment]
@fureigh sure, wasn't sure what the protocol of merging is on this team / project. Am I good to go?

**Comment by fureigh** [comment]
@jessieay I don't know. Looked for an `.about.yml` file but there isn't one. @ertzeid, you merged a PR recently — what's the protocol for this repo?

File: _pages/update-the-config-file/understanding-baseurl.md (hunk 1 above)
Comment (jessieay) [inline _pages/update-the-config-file/understanding-baseurl.md:24]: omg my first emoji reaction in GH! a momentous occassion

Comment (fureigh) [inline _pages/update-the-config-file/understanding-baseurl.md:24]: I'm so glad we're finally getting to collaborate, @jessieay.

**Comment by ertzeid** [comment]
@fureigh If you've reviewed the code, please feel free to merge and then delete the branch. I only just inherited this repo and am not quite up to speed on its processes, but all the others in the Practice Team suite follow the reviewer-does-the-merge pattern.

**Comment by fureigh** [comment]
Thanks, @ertzeid! Sounds like @jessieay gets to do the honors, then.

**Pull request merged**